src/platforms/generic_unix/main.c

```diff
@@ -31,7 +31,7 @@
 #include "globalcontext.h"
 #include "iff.h"
 #include "module.h"
-#include "platforms/generic_unix/mapped_file.h"
```

Comment:
+1

Reply:
I realize these imports should be sorted.

Reply:
(so I'm removing this file from the clang-format-ignore list)